docs/move-out-of-stock-products-to-the-end-of-a-collection/README.md

```diff
@@ -27,8 +27,11 @@ This task re-sorts your collections, beginning with the sort order of your choic
 
 ```liquid
 mechanic/user/trigger
-{% if options.run_hourly__boolean %}mechanic/scheduler/hourly{% endif %}
-{% if options.run_daily__boolean %}mechanic/scheduler/daily{% endif %}
+{% if options.run_hourly__boolean %}
+  mechanic/scheduler/hourly
+{% elsif options.run_daily__boolean %}
+  mechanic/scheduler/daily
+{% endif %}
 ```
 
 [Learn about event subscriptions in Mechanic](https://docs.usemechanic.com/article/408-subscriptions)
```

Comment:
https://learn.mechanic.dev/core/tasks/subscriptions

Update link 👆

Reply:
It looks like that link and a few others need to be updated in the task build readme template.

Reply:
Ahh :D indeed!

Reply:
Going to get this fix out there and then we'll get the auto-generated links fixed up